packages/react-native-usb/android/src/main/java/io/trezor/rnusb/ReactNativeUsbModule.kt

```diff
@@ -362,6 +377,6 @@ fun getWebUSBDevice(device: UsbDevice): WebUSBDevice {
             "serialNumber" to device.serialNumber,
             //"configuration" to device.configuration,
             //"configurations" to device.configurations,
-            "opened" to false,
+            "opened" to opened,
```

Comment:
Why not use `hasOpenedConnection(device.deviceName)` here instead of passing it as param?

Reply:
I see it's not method but standalone function, then we should make it method

Reply:
I like standalone functions more :-) but of course I can make it a method here, let's do it.

Reply:
I think it doesn't make sense to pass information manually which could only lead to bugs.

Reply:
7e3b6dec61d6956cee803793fd15e4d6e26b32de